share/completions/mkdir.fish

```diff
@@ -0,0 +1,9 @@
+complete -c mkdir -s m -l mode --description 'Set file mode (as in chmod)' -x
+complete -c mkdir -s p -l parents --description 'Make parent directories as needed'
+complete -c mkdir -s v -l verbose --description 'Print a message for each created directory'
+complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
```

Comment:
In this case, I can do something like this:

```fish
if test (uname) = "Linux"
    complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
    complete -c mkdir -l context --description 'Like -Z' -f
end
```

Or I can get rid of the file.